### PR TITLE
[user] Search form creation

### PR DESCRIPTION
**Pull request by ndufreche**
Creation of the search form with name, email, login and status input.

Removing the login input from edit user form.

And remove one console log forgotten in other commit.

### CODE REVIEW

File: src/tb/apps/user/templates/toolbar.twig [hunk 1]
```diff
@@ -27,65 +11,61 @@
...
                                 <div class="col-bb5-50">
-                                    <p>{{ 'find_users'|trans }}</p>
                                     <p class="row form-group">
-                                        <span class="col-bb5-x">{{ 'status'|trans }}</span>
+                                        <span class="col-bb5-x"><label for="login">{{ 'login'|trans }}</label></span>
                                         <span class="col-bb5-55">
-                                            <select class="form-control input-xs" name="[]">
-                                                <option value="">{{ 'all'|trans }}</option>
-                                            </select>
+                                            <input type="text" class="form-control input-xs" name="login">
```
Comment (mickaelandrieu) [inline src/tb/apps/user/templates/toolbar.twig]: I'm not sure, but the ``for`` attribute of an label refers to id attribute of an input element.

**Comment by mickaelandrieu** [comment]
A little fixup, and so on ! let's release this damn major version :dancer:

File: src/tb/apps/user/templates/toolbar.twig (hunk 1 above)
Comment (ndufreche) [inline src/tb/apps/user/templates/toolbar.twig]: I have check and yes for is a reference to the input id

**Comment by ndufreche** [comment]
I'm not for the fixup, there is the form login and I have change the edit form too.

**Pull request merged**